.circleci/cimodel/data/binary_build_data.py

```diff
@@ -39,6 +39,7 @@ def get_processor_arch_name(cuda_version):
         "3.5m",
         "3.6m",
         "3.7m",
+        "3.8m",
```

Comment:
This is really the only change in the actual change-set since the rest are just generated configurations.